lib/ts/version.ts

```diff
@@ -12,6 +12,6 @@
  * License for the specific language governing permissions and limitations
  * under the License.
  */
-export const version = "8.2.0";
+export const version = "8.2.1";
```

Comment:
This should be `8.3.0` as it's a new, non breaking feature

Reply:
Updated